diff -urN aa/gnome-desktop-2.5.91/desktop-docs/fdl/C/fdl-C.omf gnome-desktop-2.5.91/desktop-docs/fdl/C/fdl-C.omf
--- aa/gnome-desktop-2.5.91/desktop-docs/fdl/C/fdl-C.omf	2004-03-09 11:22:13.000000000 +0100
+++ gnome-desktop-2.5.91/desktop-docs/fdl/C/fdl-C.omf	2004-03-09 22:07:57.561277664 +0100
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Free Documentation License
     </title>
+    <date>
+      2000-12-16
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="fdl.xml"/>
     <language code="C"/>
+    <relation seriesid="9c6bdcc4-63b9-11d8-954e-e8ce74115f00"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.5.91/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf gnome-desktop-2.5.91/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf
--- aa/gnome-desktop-2.5.91/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf	2004-03-09 11:17:59.000000000 +0100
+++ gnome-desktop-2.5.91/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf	2004-03-09 22:09:39.285813184 +0100
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Alexander Kirillov
+    </creator>
     <title>
       Reporting GNOME Bugs and Other Feedback
     </title>
+    <date>
+      2001-04-19
+    </date>
     <subject category="GNOME|Core Desktop"/>
-    <format mime="text/sgml"/>
-    <identifier url="gnome-feedback.sgml"/>
+    <format mime="text/xml"/>
+    <identifier url="gnome-feedback.xml"/>
     <language code="C"/>
+    <relation seriesid="0ad1f6bc-63ba-11d8-9f2a-c089f27f14d9"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.5.91/desktop-docs/gpl/C/gpl-C.omf gnome-desktop-2.5.91/desktop-docs/gpl/C/gpl-C.omf
--- aa/gnome-desktop-2.5.91/desktop-docs/gpl/C/gpl-C.omf	2004-03-09 11:18:34.000000000 +0100
+++ gnome-desktop-2.5.91/desktop-docs/gpl/C/gpl-C.omf	2004-03-09 22:11:01.648292184 +0100
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU General Public License
     </title>
+    <date>
+      2000-16-12
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="gpl.xml"/>
     <language code="C"/>
+    <relation seriesid="3276c766-63b9-11d8-9049-a39b54b0da33"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.5.91/desktop-docs/lgpl/C/lgpl-C.omf gnome-desktop-2.5.91/desktop-docs/lgpl/C/lgpl-C.omf
--- aa/gnome-desktop-2.5.91/desktop-docs/lgpl/C/lgpl-C.omf	2004-03-09 11:19:11.000000000 +0100
+++ gnome-desktop-2.5.91/desktop-docs/lgpl/C/lgpl-C.omf	2004-03-09 22:12:40.759225032 +0100
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Lesser General Public License
     </title>
+    <date>
+      2001-02-05
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="lgpl.xml"/>
     <language code="C"/>
+    <relation seriesid="d9417686-63b9-11d8-87df-cbf27b7c92ff"/>
   </resource>
 </omf>
