--- gnome-desktop-2.3.7/libgnome-desktop/gnome-desktop-item.c.crystalsvg	2003-08-01 16:51:49.000000000 +0200
+++ gnome-desktop-2.3.7/libgnome-desktop/gnome-desktop-item.c	2003-08-28 19:46:11.000000000 +0200
@@ -2343,6 +2343,10 @@
 static GSList *hicolor_kde_32 = NULL;
 static GSList *hicolor_kde_22 = NULL;
 static GSList *hicolor_kde_16 = NULL;
+static GSList *crystalsvg_kde_48 = NULL;
+static GSList *crystalsvg_kde_32 = NULL;
+static GSList *crystalsvg_kde_22 = NULL;
+static GSList *crystalsvg_kde_16 = NULL;
 /* XXX: maybe we don't care about locolor
 static GSList *locolor_kde_48 = NULL;
 static GSList *locolor_kde_32 = NULL;
@@ -2401,6 +2405,11 @@
 	ADD_DIRS (hicolor, 22);
 	ADD_DIRS (hicolor, 16);
 
+	ADD_DIRS (crystalsvg, 48);
+	ADD_DIRS (crystalsvg, 32);
+	ADD_DIRS (crystalsvg, 22);
+	ADD_DIRS (crystalsvg, 16);
+
 /* XXX: maybe we don't care about locolor
 	ADD_DIRS (locolor, 48);
 	ADD_DIRS (locolor, 32);
@@ -2420,36 +2429,68 @@
 
 	if (size > 32) {
 		/* 48-inf */
-		list = g_slist_concat (g_slist_copy (hicolor_kde_48),
+		list = g_slist_concat (g_slist_copy (crystalsvg_kde_48),
+				       g_slist_copy (hicolor_kde_48));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_32));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_32));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_22));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_22));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_16));
+		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_16));
 	} else if (size > 22) {
 		/* 23-32 */
-		list = g_slist_concat (g_slist_copy (hicolor_kde_32),
+		list = g_slist_concat (g_slist_copy (crystalsvg_kde_32),
+				       g_slist_copy (hicolor_kde_32));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_48));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_48));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_22));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_22));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_16));
+		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_16));
 	} else if (size > 16) {
 		/* 17-22 */
+		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_22));
 		list = g_slist_concat (g_slist_copy (hicolor_kde_22),
 				       g_slist_copy (hicolor_kde_32));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_32));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_48));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_48));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_16));
+		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_16));
 	} else {
 		/* 1-16 */
-		list = g_slist_concat (g_slist_copy (hicolor_kde_16),
+		list = g_slist_concat (g_slist_copy (crystalsvg_kde_16),
+				       g_slist_copy (hicolor_kde_16));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_22));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_22));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_32));
 		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_32));
+		list = g_slist_concat (list,
 				       g_slist_copy (hicolor_kde_48));
+		list = g_slist_concat (list,
+				       g_slist_copy (crystalsvg_kde_48));
 	}
 
 	list = g_slist_append (list, kde_icondir);
