diff -urN aa/gnome-desktop-2.6.2/desktop-docs/fdl/C/fdl-C.omf gnome-desktop-2.6.2/desktop-docs/fdl/C/fdl-C.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/fdl/C/fdl-C.omf	2004-03-09 11:29:13.000000000 +0100
+++ gnome-desktop-2.6.2/desktop-docs/fdl/C/fdl-C.omf	2004-06-14 23:46:23.349267880 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Free Documentation License
     </title>
+    <date>
+      2000-12-16
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="fdl.xml"/>
     <language code="C"/>
+    <relation seriesid="9c6bdcc4-63b9-11d8-954e-e8ce74115f00"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/fdl/eu/fdl-eu.omf gnome-desktop-2.6.2/desktop-docs/fdl/eu/fdl-eu.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/fdl/eu/fdl-eu.omf	2004-04-26 19:00:02.000000000 +0200
+++ gnome-desktop-2.6.2/desktop-docs/fdl/eu/fdl-eu.omf	2004-06-14 23:51:30.240613304 +0200
@@ -1,14 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Free Documentation License
     </title>
-    <subject>
-      <category>General|Licenses</category>
-    </subject>
+    <date>
+      2000-12-16
+    </date>
+    <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="fdl.xml"/>
     <language code="eu"/>
+    <relation seriesid="9c6bdcc4-63b9-11d8-954e-e8ce74115f00"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf gnome-desktop-2.6.2/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf	2004-03-09 11:29:14.000000000 +0100
+++ gnome-desktop-2.6.2/desktop-docs/gnome-feedback/C/gnome-feedback-C.omf	2004-06-14 23:46:23.349267880 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Alexander Kirillov
+    </creator>
     <title>
       Reporting GNOME Bugs and Other Feedback
     </title>
+    <date>
+      2001-04-19
+    </date>
     <subject category="GNOME|Core Desktop"/>
-    <format mime="text/sgml"/>
-    <identifier url="gnome-feedback.sgml"/>
+    <format mime="text/xml"/>
+    <identifier url="gnome-feedback.xml"/>
     <language code="C"/>
+    <relation seriesid="0ad1f6bc-63ba-11d8-9f2a-c089f27f14d9"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/gnome-feedback/eu/gnome-feedback-eu.omf gnome-desktop-2.6.2/desktop-docs/gnome-feedback/eu/gnome-feedback-eu.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/gnome-feedback/eu/gnome-feedback-eu.omf	2004-04-26 19:00:02.000000000 +0200
+++ gnome-desktop-2.6.2/desktop-docs/gnome-feedback/eu/gnome-feedback-eu.omf	2004-06-15 00:02:01.826597640 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Alexander Kirillov
+    </creator>
     <title>
       Reporting GNOME Bugs and Other Feedback
     </title>
+    <date>
+      2001-04-19
+    </date>
     <subject category="GNOME|Core Desktop"/>
     <format mime="text/sgml"/>
     <identifier url="gnome-feedback.sgml"/>
     <language code="eu"/>
+    <relation seriesid="0ad1f6bc-63ba-11d8-9f2a-c089f27f14d9"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/gpl/C/gpl-C.omf gnome-desktop-2.6.2/desktop-docs/gpl/C/gpl-C.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/gpl/C/gpl-C.omf	2004-03-09 11:29:15.000000000 +0100
+++ gnome-desktop-2.6.2/desktop-docs/gpl/C/gpl-C.omf	2004-06-14 23:46:23.350267728 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU General Public License
     </title>
+    <date>
+      2000-16-12
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="gpl.xml"/>
     <language code="C"/>
+    <relation seriesid="3276c766-63b9-11d8-9049-a39b54b0da33"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/gpl/eu/gpl-eu.omf gnome-desktop-2.6.2/desktop-docs/gpl/eu/gpl-eu.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/gpl/eu/gpl-eu.omf	2004-04-26 19:00:02.000000000 +0200
+++ gnome-desktop-2.6.2/desktop-docs/gpl/eu/gpl-eu.omf	2004-06-15 00:08:18.879276920 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU General Public License
     </title>
+    <date>
+      2000-16-12
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="gpl.xml"/>
     <language code="eu"/>
+    <relation seriesid="3276c766-63b9-11d8-9049-a39b54b0da33"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/lgpl/C/lgpl-C.omf gnome-desktop-2.6.2/desktop-docs/lgpl/C/lgpl-C.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/lgpl/C/lgpl-C.omf	2004-03-09 11:29:15.000000000 +0100
+++ gnome-desktop-2.6.2/desktop-docs/lgpl/C/lgpl-C.omf	2004-06-14 23:46:23.350267728 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Lesser General Public License
     </title>
+    <date>
+      2001-02-05
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="lgpl.xml"/>
     <language code="C"/>
+    <relation seriesid="d9417686-63b9-11d8-87df-cbf27b7c92ff"/>
   </resource>
 </omf>
diff -urN aa/gnome-desktop-2.6.2/desktop-docs/lgpl/eu/lgpl-eu.omf gnome-desktop-2.6.2/desktop-docs/lgpl/eu/lgpl-eu.omf
--- aa/gnome-desktop-2.6.2/desktop-docs/lgpl/eu/lgpl-eu.omf	2004-04-26 19:00:03.000000000 +0200
+++ gnome-desktop-2.6.2/desktop-docs/lgpl/eu/lgpl-eu.omf	2004-06-15 00:12:40.103564792 +0200
@@ -1,12 +1,20 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator>
+      Free Software Foundation
+    </creator>
     <title>
       GNU Lesser General Public License
     </title>
+    <date>
+      2001-02-05
+    </date>
     <subject category="General|Licenses"/>
     <format mime="text/xml"/>
     <identifier url="lgpl.xml"/>
     <language code="eu"/>
+    <relation seriesid="d9417686-63b9-11d8-87df-cbf27b7c92ff"/>
   </resource>
 </omf>
